src/components/auth/RegisterParent.vue

```diff
@@ -169,11 +170,18 @@
         </div>
       </form>
     </div>
+    <div class="flex align-items-center justify-content-center py-3 mt-3 border-top-1 border-gray-200">
+      <RouterLink :to="APP_ROUTES.SIGN_IN" class="text-sm text-blue-600 hover:text-blue-800">
```

Comment:
Let's adjust the text colour to match the overall ROAR colour scheme and add the padding to the link itself so that the click surface is larger. Even though ROAR is not really responsive/mobile compatible, we should still keep these things in mind as we add/refactor features. 

```suggestion
      <RouterLink :to="APP_ROUTES.SIGN_IN" class="text-sm transition text-gray-600 hover:text-gray-800 p-3">
```